Gcp-vpc-network-challenge-lab-guide.html

```
@@ -0,0 +1,1677 @@
+                                    <td>
+                                        組織またはリージョン単位でルールを一元管理できる階層型ファイアウォールポリシーやネットワークファイアウォールポリシーを活用する
+                                    </td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+
+                    <h3>事前チェックリスト（本番移行前）</h3>
+                    <ul class="checklist">
+                        <li>
+                            <input type="checkbox" disabled /> SSH/RDPの送信元範囲を
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)